eng/tools/generator/autorest/model/changelog.go

```diff
@@ -419,3 +440,24 @@ func removePattern(funcName string, returnValue string) string {
 
 	return fmt.Sprintf("%s.%s", before, after)
 }
+
+func typeToAny(b *report.BreakingChanges) []string {
+	if b.IsEmpty() {
+		return nil
+	}
+
+	var items []string
+	if len(b.Structs) > 0 {
+		for _, k := range sortChangeItem(b.Structs) {
+			v := b.Structs[k]
+			for f, d := range v.Fields {
+				if strings.Contains(d.To, "any") {
```

Comment:
I'm a little bit struggling about this condition. For most situation, is could be right, but I wonder if there is any corner case that will break it, e.g., `map[A]string` -> `map[B]any`. To be safe, perhaps we only support exact equal at first.